app/src/main/java/com/example/perfectweatherallyear/utils/ForecastService.kt

```diff
@@ -38,16 +38,20 @@ class ForecastService : Service() {
         return null
     }
 
+    override fun onCreate() {
+        super.onCreate()
+        notification = NotificationHandler.createAndPostNotification(application, applicationContext, getString(R.string.notification_text))
```

Comment:
Who don't need to post notification at this point. It will be automatically posted in 
startForeground(START_NOTIFICATION_SERVICE_ID, notification)